accessing.md

```diff
@@ -21,7 +21,7 @@ func main() {
 
 In the example shown, we're illustrating several things:
 
-1. The first argument to `sql.Open` is the driver name. This is the string that the driver used to register itself with `database/sql`, and is conventionally the same as the package name to avoid confusion.
+1. The first argument to `sql.Open` is the driver name. This is the string that the driver used to register itself with `database/sql`, and is conventionally the same as the package name to avoid confusion. It's `mysql` for [MySQL Server](github.com/go-sql-driver/mysql). Or `sqlite3` if you use [SQLite database engine](https://github.com/mattn/go-sqlite3), `postgres` for [PostgreSQL server](https://github.com/lib/pq).
```

Comment:
The first link is missing `https://`. Also, I think it is more accurate to write the package paths, e.g.
> `mysql` for [github.com/go-sql-driver/mysql](https://github.com/go-sql-driver/mysql), `sqlite3` for [github.com/mattn/go-sqlite3](https://github.com/mattn/go-sqlite3), ...

since there can be different drivers for the same database.